nms/packages/magmalte/fbc_js_core/ui/host/Organizations.js

```diff
@@ -231,13 +147,24 @@ function Organizations(props: Props) {
     [classes.index],
   );
 
-  const renderLinkColumn = useCallback(rowData => {
-    return (
-      <Link href={rowData.portalLink}>
-        {`Visit ${rowData.name} Organization Portal`}
-      </Link>
-    );
-  }, []);
+  const renderLinkColumn = useCallback(
+    rowData => {
+      return (
+        <Link
+          href={rowData.portalLink}
+          variant="body2"
+          ref={
+            organizations?.length === rowData.tableData?.index + 1 &&
+            organizations?.length < 3
+              ? linkHelperRef
+              : null
+          }>
+          {`Visit ${rowData.name} Organization Portal`}
```

Comment:
I think we'd want a special case here for the "Host" organization, to show something like

`'Visit Host Portal'` instead of `'Visit Host Organization Portal'`